AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/QtPyWidgetContainer.py

```diff
@@ -0,0 +1,52 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Object to house all the Qt Objects used when testing and manipulating the O3DE UI
+"""
+import pyside_utils
+import editor_python_test_tools.QtPyScriptCanvasEditor as QtPyScriptCanvasEditor
+import editor_python_test_tools.QtPyScriptCanvasVariableManager as QtPyScriptCanvasVariableManager
+
+
+class QtPyWidgetContainer:
+    """
+    Container class for QtPy objects used in testing/manipulating the UI
```

Comment:
Why does this need to hold state? What is this simplifying?

Reply:
script canvas tests often need multiple Qt objects. We want to be able to tell a single object to set up all the script canvas qt objects in a single call and keep a handle on them for later test steps.

Reply:
container class has been removed